src/components/regle.js

```diff
@@ -0,0 +1,92 @@
+import {fr} from '@codegouvfr/react-dsfr'
+import {Tooltip} from '@codegouvfr/react-dsfr/Tooltip'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import {
+  Box, Typography, Accordion, AccordionSummary,
+  AccordionDetails
+} from '@mui/material'
+
+import {formatDateRange, formatFullDateFr} from '@/lib/format-date.js'
+import {getParametreInfo, getRegleContrainte} from '@/lib/regles.js'
+import {getDocumentURL} from '@/lib/urls.js'
+
+const InfoRow = ({label, value, description}) => (
+  <Box className='flex justify-between'>
+    <Typography fontWeight='medium' className='fr-text--sm'>{label}</Typography>
+    <Box className='flex gap-1'>
+      <Typography fontWeight='light' className='fr-text--sm'>{value || '-'}</Typography>
+      {description && <Tooltip title={description} />}
+    </Box>
+  </Box>
+)
+
+const RegleHeader = ({parametre, debutValidite, finValidite, debutPeriode, finPeriode, unite, valeur, contrainte}) => {
+  const {label, icon} = getParametreInfo(parametre)
+
+  return (
+    <Box className='flex flex-col w-full gap-4'>
+      <Box className='flex items-end align-middle gap-1'>
+        <span style={{color: fr.colors.decisions.text.label.blueFrance.default}}>
+          {icon}
+        </span>
+        <Typography fontWeight='bold'>{label} :</Typography>
```

Comment:
Valentin à demandé (2x) de retirer les `:`
```suggestion
        <Typography fontWeight='bold'>{label}</Typography>
```

Reply:
My bad, j'avais retenu que ça ne concernait que les dates et le contenu du dropdown 🙏🏻